index.js

```diff
@@ -0,0 +1,32 @@
+import {
```

Comment:
What’s the reason for two files in the root of the project?

- Might it make sense to move the code in `index.js` to `server.js`, and re-export things from it in `index.js`?
- Instead of multiple files in `/`, would a `lib/` folder help, which can more easily be split into several modules?